streaming/rdm_atomic.c

```diff
@@ -228,13 +228,10 @@ static inline int execute_base_atomic_op(enum fi_op op)
 {
 	int ret;
 
-	ret = fi_atomic(ep, buf, 1, mr_desc, remote_fi_addr, remote.addr,
-		       	remote.key, datatype, op, &fi_ctx_atomic);
-	if (ret) {
-		FT_PRINTERR("fi_atomic", ret);
-	} else {
-		ret = ft_get_tx_comp(++tx_seq);
-	}
+	ret = ft_post_atomic(FT_ATOMIC_BASE, ep, NULL, NULL, NULL, NULL,
+			     &remote, datatype, op, &fi_ctx_atomic);
+
```

Comment:
We lose the return value here and below.

Reply:
Oops. My b. Updated.